### PR TITLE
New checkbox design

### PR DESCRIPTION
**Pull request by kathleenAMA**
This is adding in the new checkbox design to apps. Each app will need to be looked at to make sure that they dont have checkbox designs overwritten. 

:art:

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:hankey: Can you enter a description of the purpose of this PR?

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss [hunk 1]
```diff
@@ -130,31 +141,65 @@ input[type="radio"]:checked + label{
...
+  span:first-of-type{
+    background: $brand-blue-light;
+  }
 }
 
 [type="checkbox"]:checked + label:after{
   margin-left: $base-margin*6;
 }
 
+@keyframes checkbox{
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: Is there a way I can see this in action?

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss [hunk 2]
```diff
@@ -130,31 +141,65 @@ input[type="radio"]:checked + label{
...
+  0%{
+   height: 0;
+   width: 0;
+   }
+
+  25%{
+    height: 0px;
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: Is this needed if it's the same as the previous %? Or would it scale towards the height defined at 50% (so 8.5px at 25%)? 
And I think `0px` should just be `0`, by the style rules.

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss [hunk 3]
```diff
@@ -130,31 +141,65 @@ input[type="radio"]:checked + label{
...
+    width: 8px;
+  }
+
+  50%{
+    height: 17px;
+    width: 8px;
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: Is this width definition needed, if it's the same as the last step and there's no higher-% keyframe defined to change it further?

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss (hunk 1 above)
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: It's an animation so you'd have to pull the branch down to see it.

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss (hunk 2 above)
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: Yes, the height needs to be defined again at this keyframe for a smooth animation.

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss (hunk 3 above)
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: I like to define both if they are the only things changing so it's easier to see the changes to both of them. I think some browsers also render these differently and need both defined.

**Comment by kathleenAMA** [comment]
:wave:

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss (hunk 3 above)
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: :+1: Sounds like a good approach.

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss (hunk 1 above)
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: No animated gif screen capture?

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Comment by kathleenAMA** [comment]
:wave:  I need palettes on this too

**Comment by ryanjones** [comment]
GIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIF

**Comment by MathieuGilbert** [comment]
:+1: https://itunes.apple.com/ca/app/gifgrabber/id668208984?mt=12

**Comment by MathieuGilbert** [comment]
:art: :art: :art: I guess the elephants should have been these instead.

**Comment by ryanjones** [comment]
GIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIFGIF GIF GIF

**Comment by kathleenAMA** [comment]
......
Here is your gif. 
checkboxes

**Comment by ryanjones** [comment]
my hero. Thanks :)!

**Comment by kathleenAMA** [comment]
:wave: so I wasnt up to date with master but it was still allowing me to merge....I  think that's a bug? I also didnt bump the gem version number

**Comment by MathieuGilbert** [comment]
Good catch on the gem version! I think I fixed the merge protection:

image

**Comment by MathieuGilbert** [comment]
:art: :art: :art: 
:elephant: :elephant: :elephant:

**Pull request merged**